com.zsmartsystems.zigbee/src/test/java/com/zsmartsystems/zigbee/zcl/ZclResponseMatcherTest.java

```diff
@@ -61,7 +62,7 @@ public void testNonMatchingTransactionIds() {
     @Test
     public void testZdoResponse() {
         ZclCommand request = createOnCommand(new ZigBeeEndpointAddress(0,1), new ZigBeeEndpointAddress(1234,5), 5);
-        ZdoCommand response = new DeviceAnnounce();
+        ZdoCommand response = new DeviceAnnounce(123, new IeeeAddress("1234"), null);
```

Comment:
Just for correctness, can you make the IeeeAddress argument 16 characters long so that it's a valid address.